packages/eventsub-base/src/subscriptions/EventSubChannelCharityCampaignProgressSubscription.ts

```diff
@@ -0,0 +1,40 @@
+import type { HelixEventSubSubscription } from '@twurple/api';
+import { rtfm } from '@twurple/common';
+import type { EventSubChannelCharityCampaignProgressEventData } from '../events/EventSubChannelCharityCampaignProgressEvent';
+import { EventSubChannelCharityCampaignProgressEvent } from '../events/EventSubChannelCharityCampaignProgressEvent';
+import type { EventSubBase } from '../EventSubBase';
+import { EventSubSubscription } from './EventSubSubscription';
+
+/**
+ * @private
+ * @beta
```

Comment:
`@private` and `@beta` is a useless combination. Private APIs do not appear in the documentation and can be changed at will anyway, so dropping the `@beta` is cleaner.

(Also applies to the other new events)